### DISCUSSION TITLE
emoji

### DISCUSSION THREAD

**Discussion by wodaixin** [mmistakes/minimal-mistakes#3027] [Q&A] [answered]
## Motivation
Somewhere emoji don't include , but docs include  [plugins](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_config.yml)

## Suggestion
e.g. the Installation document https://mmistakes.github.io/minimal-mistakes/docs/installation/#install-dependencies 

The following plugins are automatically loaded

should be add

``` ruby
    gem "jemoji"
```

**Comment by mmistakes** [comment, marked as answer]
It's up to the user of the theme to install that plugin if they want. I don't see a need to force another dependency especially since it's [installed by default if hosting on GitHub Pages](https://pages.github.com/versions/) as a remote theme.
